### DISCUSSION TITLE
Social buttons

### DISCUSSION THREAD

**Discussion by hamzaelmaghari** [saadeghi/daisyui#718] [Ideas / feature requests]
It would be better if social buttons integrated.
With all buttons states.

Is there anything I can do to grab the buttons states and styles to make my own social buttons?

**Comment by saadeghi** [comment]
You just need to add a logo and the brand colors. Example: https://codepen.io/saadeghi/pen/popxGzO?editors=1000

**Comment by hamzaelmaghari** [comment]
Thanks @saadeghi for answering my question.
But as you can see we still have an issue of states such as focus, active, hover, etc...

We need a mecanism that we can use to add a layer over Daisyui styles.

I hope see this feature in the future 👍

**Reply by saadeghi** [reply]
It's not an "issue".  
A "login with facebook" button is a single button with a specific color. I don't think there should be a reusable `btn-facebook` class just for that single button. If it's a one-time thing, it's totally okay to use Tailwind CSS arbitrary utility classes.
